package.json

```diff
@@ -4,7 +4,7 @@
   "description": "",
   "main": "index.js",
   "scripts": {
-    "testrpc": "ganache-cli --mnemonic 'clock radar mass judge dismiss just intact mind resemble fringe diary casino' --gasLimit 13000000",
+    "testrpc": "ganache-cli --hardfork istanbul -p 7545 --networkId 5777 --mnemonic 'clock radar mass judge dismiss just intact mind resemble fringe diary casino' --gasLimit 13000000 --gasPrice 0",
```

Comment:
We use port 8545 for child chain, so let that be default.
Usually use the network id of 15001.
Remove `--gasPrice 0`.

Reply:
okay i'll change

Reply:
done